payments-core/src/main/java/com/stripe/android/payments/paymentlauncher/PaymentLauncherFactory.kt

```diff
@@ -18,29 +18,32 @@ import kotlinx.coroutines.Dispatchers
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 class PaymentLauncherFactory(
     private val context: Context,
-    private val hostActivityLauncher: ActivityResultLauncher<PaymentLauncherContract.Args>
+    private val hostActivityLauncher: ActivityResultLauncher<PaymentLauncherContract.Args>,
+    private val statusBarColor: () -> Int?,
```

Comment:
Why does this need to be a lambda?

Reply:
I thought I encountered a situation where the value wasn’t set immediately, but now I can’t reproduce it. I’ve changed it back to a value instead of a provider function.